app/init/launch.ts

```diff
@@ -113,6 +116,8 @@ const launchApp = async (props: LaunchProps) => {
         serverUrl = await getActiveServerUrl();
     }
 
+    await cleanupEphemeralPosts();
```

Comment:
I wonder if we should even await.  Could we just fire and forget?

Reply:
Right. It makes more sense not to let this slow down the app launch 👍